### PR TITLE
Chapter1 homework 5

### PR DESCRIPTION
**Pull request by EMonfort**
Added code for Chapter 1 Homework 5

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -29,8 +59,16 @@ public static void Reverse(int[] array)
         /// <returns>A new array with the last element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveLast(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:60]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -39,8 +77,16 @@ public static int[] RemoveLast(int[] array)
         /// <returns>A new array with the first element removed. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveFirst(int[] array)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:74]: Reuse RemoveAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -63,8 +129,18 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:117]: Reuse InsertAt

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -75,8 +151,18 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:128]: Reuse InsertAt

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 4 above)
Comment (EMonfort) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:128]: Made the requested changes

**Review by Almantask — APPROVED** [review]

**Pull request closed without merging**